src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1262,11 +1290,19 @@ class AbstractUtxoCoin extends BaseCoin {
 
       const queries = [];
 
-      _.forEach(AbstractUtxoCoin.AddressTypes, function(addressType) {
+      _.forEach(Object.keys(Codes.UnspentTypeTcomb.meta.map), function(addressType) {
```

Comment:
Aren't key and value in the UnspentTypes identical anyway?

Reply:
Not sure your point here. Without `Object.keys` we would be looping over a map.

Reply:
Just looked complicated and I thought we could just get to a list of the keys or values easier.